cogs/tagging.py

```diff
@@ -21,14 +21,14 @@ async def tag(self, ctx, name=None):
                 return
             emb = discord.Embed()
             emb.set_footer(text=f'\t\t\t\t\t\t\tTimestamp: {result["date"]}',
-                           icon_url="https://cdn.discordapp.com/attachments/336577284322623499/683028692133216300/ac6e275e1f638f4e19af408d8440e1d1.png")
+                           icon_url='https://avatars1.githubusercontent.com/u/42101452?s=200&v=4')
```

Comment:
What is this link?

Reply:
Same icon, just shorter link. Exists in `libs.yml` for quite a long time.